### DISCUSSION TITLE
Duplicated resource reconcile event after first app launch

### DISCUSSION THREAD

**Discussion by vvitkovsky** [dotnet/dotnet-operator-sdk#571] [Q&A]
Hi gents, 

I have a strange situation - when I start app first time and resource controller is started to work I receive two ReconcileAsync calls for the same resource, but I get two different versions of the same resource:

ReconcileAsync:: Version: 399663
ReconcileAsync:: Version: 399662

the order can be different, but in general I always get two versions on the controller start.
Is it bug or feature? If feature, can this behavior be switched off so I will receive only latest version of resource once?  

Best regards,
Victor

**Comment by buehler** [comment]
Hey @vvitkovsky 

This should not happen to be honest. Do you modify the resource in some way?

**Comment by vvitkovsky** [comment]
Hi Christoph, 

No, I am using official KubeOps nuget package 7.3.0.

I have only one crd object:
![image](https://github.com/buehler/dotnet-operator-sdk/assets/24652637/610da849-503e-4b27-9066-934a601a4738)

But when I start my app I see those two reconcile calls for this one object, and versions of it are different:
![image](https://github.com/buehler/dotnet-operator-sdk/assets/24652637/878446aa-6bb8-4f33-b09f-b4ba1b0953e1)

Best regards,
Victor
